buildspec_windows_verify_ssm_parameters.yml

```diff
@@ -0,0 +1,16 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.x
+    commands:
+      - echo "Verifies the SSM parameters for Windows"
+      - yum update -y && yum upgrade -y
+      - yum install jq -y
+  build:
+    commands:
+      # Validates that the SSM parameters are published as required.
+      # The required variables would be passed on via CodeBuild Environment variables.
+      - cd ./scripts
+      - './verify_windows_ssm_parameters.sh'
```

Comment:
why separate buildspecs and projects for publishing and verifying?

Reply:
So for the `Build-Images` phase of our pipeline, we did need separate build projects (One for build and one for manifest creation) and consequently, the verify phase was separated. In short, we have the following-
- Build images stage
- Build manifest stage
- Verify image stage

Same goes for `Publish Images` phase as well since we need to create Windows instances to pull and push the images.

To ensure that we had similar semantics across the pipeline, I created SSM stage in two parts as well-
- Publish SSM Parameters
- Verify SSM Parameters